### ISSUE TITLE
Invalid paths on Windows in manual tests

### ISSUE THREAD

**Issue by pomek** [ckeditor/ckeditor5-dev-tests#30]
You cannot use `gulp test:manual` on Windows because it throws an error:

```
[16:01:25] Using gulpfile C:\cks\ckeditor5\gulpfile.js
[16:01:25] Starting 'test:manual'...
[16:01:25] Compiling the editor...
Error: watch C:\cks\ckeditor5\build\.manual-tests\C:\cks\ckeditor5\build\.manual
-tests\tests\autoformat\manual\autoformat.html ENOENT
   at exports._errnoException (util.js:953:11)
   at FSWatcher.start (fs.js:1423:19)
   at Object.fs.watch (fs.js:1450:11)
   at Object._watchFiles (C:\cks\ckeditor5\node_modules\@ckeditor\ckeditor5-dev
-tests\lib\utils.js:373:7)
   at utils._getManualTestPaths.map.forEach (C:\cks\ckeditor5\node_modules\@cke
ditor\ckeditor5-dev-tests\lib\tasks.js:197:12)
   at Array.forEach (native)
   at Object.compileViews (C:\cks\ckeditor5\node_modules\@ckeditor\ckeditor5-de
v-tests\lib\tasks.js:192:6)
   at Timeout.checkWaitUntil [as _onTimeout] (C:\cks\ckeditor5\node_modules\@ck
editor\ckeditor5-dev-tests\lib\tasks.js:251:18)
   at tryOnTimeout (timers.js:224:11)
   at Timer.listOnTimeout (timers.js:198:5)
```

Steps to reproduce:

1. Use `gulp test:manual` on Windows environment.

**Comment by pomek** [comment]
Fix for it is simple.

In the documentation of [`node-glob`](https://github.com/isaacs/node-glob#windows) I found we should always use a slash `/` as a directory separator.

Glob does not understand backslash and returns paths separated by `/`.

On Windows environment, we have to replace `/` to `\` manually.

**Comment by pomek** [comment]
Fixed in #33.
